include/uacpi/acpi.h

```diff
@@ -1003,6 +1003,66 @@ UACPI_PACKED(struct acpi_ecdt {
 })
 UACPI_EXPECT_SIZEOF(struct acpi_ecdt, 65);
 
+#define ACPI_DBG2_TYPE_SERIAL 0x8000
+#define ACPI_DBG2_TYPE_1394 0x8001
+#define ACPI_DBG2_TYPE_USB 0x8002
+#define ACPI_DBG2_TYPE_NET 0x8003
+
+#define ACPI_DBG2_SUBTYPE_SERIAL_NS16550 0x0
+#define ACPI_DBG2_SUBTYPE_SERIAL_NS16550_DBGP1 0x1
+#define ACPI_DBG2_SUBTYPE_SERIAL_PL011 0x3
```

Comment:
If you look at other defines usually I try to mention which field it belongs to, for example `// acpi_dbg2_dbg_device_info->port_subtype`